research_spec.md

```diff
@@ -36,7 +36,7 @@ For a round that will start its active phase at time `t`, the round proceeds lik
 
 - At any time between t-`AuctionClosingSeconds` and t, the autonomous auctioneer closes the auction by submitting a transaction to the auction contract, as described above. The autonomous auctioneer is trusted to follow the instructions specified above. This may designate a party as the express lane controller for the round, and if so it will deduct funds from that party's deposit account and distribute or burn those funds.
 
-- At any time between the designation of the express lane controller and the end of the round (t+`RoundDurationSeconds`), the express lane controller, or an alternative `transferrer` if set, can call the auction contract to transfer the express lane controller right for the round to a different address. By doing this, the giver loses all of its own rights in the round. The express lane controller can set this alternative `transferrer` by calling `setTransferrer` function at any time, this `transferrer` will have the ability to transfer the rights of any current or future round controlled by that express lane controller.
+- At any time between the designation of the express lane controller for the round starting at t and the end of the round (t+`RoundDurationSeconds`), the express lane controller, or an alternative `transferrer` if set, can call the `transferExpressLaneController` on the auction contract to transfer the express lane controller right for the round to a new address. By calling `transferExpressLaneController` and transferring express lane controller rights the current controller loses all of its own rights in the round. The transferee receives express lane controller rights from the moment they are transferred, as well as the right to transfer to another address, either directly or by an alternative transferrer if set. Each express lane controller can set their own alternative `transferrer` by calling `setTransferrer` function at any time, this `transferrer` will have the ability to transfer the rights of any current or future round controlled by that express lane controller. Along with the new transferrer address, an optional round number is provided when calling `setTransferrer` which specifies when the transferrer can next be updated. If a round number is not provided the transferrer can be updated at any time.
```

Comment:
> round starting at t and the end of the round (t+`RoundDurationSeconds`)

Shouldn't this be `t - AuctionClosingSeconds` and `t + RoundDurationSeconds`

Reply:
We cant give transferral rights until we know who the express lane controller is - which may happen at any time after t-`AuctionClosingSeconds` and t

Reply:
Oh ya. I meant to say when the auction is resolved to t + RoundDurationSeconds. I'm not sure how to phrase it though

Reply:
Yeh, clearer to write "auction resolved" than "designation of express lane controller". I'll update.